pychromecast/socket_client.py

```diff
@@ -1133,7 +1142,12 @@ def new_socket() -> socket.socket:
     Try to set SO_REUSEPORT for BSD-flavored systems if it's an option.
     Catches errors if not.
     """
-    new_sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        new_sock = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
+        new_sock.setsockopt(socket.IPPROTO_IPV6, socket.IPV6_V6ONLY, 0)
+    except OSError:
+        new_sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
It would be nice to add a comment here that some systems will have IPv6 disabled in the kernel (or otherwise broken) so we need to fallback to AF_INET